source/_posts/2016-09-14-what_the_tag_series_ep1_document_object.markdown

```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title: "What the tag? Episode 1 - The Document object"
+category: html
+comments: true
+author:
+  name: 'Federico Kauffman'
+  email: 'federico@wyeworks.com'
+  twitter_handle: 'fedekauffman'
+  github_handle: 'fedekau'
+  image: '/images/team/federico-kauffman.jpg'
+  description: "Developer working with Ruby and Ember, passionate about technology, robotics and crossfit (Yes, I\'am crazy)."
+---
+
+Twenty three years ago as I was learning to walk something more important was happening, something that has been part of the technology we use every day, something that pretty much brought us where we are (in terms of web development), that was HTML's birth!
+
+In this series we will dig into the modern HTML's DNA (or as the people at [WHATWG](https://whatwg.org/) call it, [The Living Standard](https://html.spec.whatwg.org/multipage/dom.html)), we will look for things that are not usual to see or people just don't know because anyone told them, and we will try to explore them.
+
+<!--more-->
+
+I've been working with HTML for 2 years, but I am pretty sure I am going to learn as much as many people with this adventure, so don't worry, if you just started coding and you feel that your sites still look like [the world's worst website ever](http://www.theworldsworstwebsiteever.com/) we will learn, and that's the important thing.
+
+# The Document object
```

Comment:
Hmm I think I see what you're trying to do, let me re-read all and give a more precise feedback.